packages/flutter_driver/lib/src/extension.dart

```diff
@@ -73,15 +75,16 @@ class _FlutterDriverExtension {
     });
 
     _commandDeserializers.addAll(<String, CommandDeserializerCallback>{
-      'get_health': (Map<String, dynamic> json) => new GetHealth.deserialize(json),
-      'get_render_tree': (Map<String, dynamic> json) => new GetRenderTree.deserialize(json),
-      'tap': (Map<String, dynamic> json) => new Tap.deserialize(json),
-      'get_text': (Map<String, dynamic> json) => new GetText.deserialize(json),
-      'scroll': (Map<String, dynamic> json) => new Scroll.deserialize(json),
-      'scrollIntoView': (Map<String, dynamic> json) => new ScrollIntoView.deserialize(json),
-      'setInputText': (Map<String, dynamic> json) => new SetInputText.deserialize(json),
-      'submitInputText': (Map<String, dynamic> json) => new SubmitInputText.deserialize(json),
-      'waitFor': (Map<String, dynamic> json) => new WaitFor.deserialize(json),
+      'get_health': (Map<String, String> json) => new GetHealth.deserialize(json),
```

Comment:
"json" is the data, which is exchanged between the test and the extension via invokeExtension [1] and it is defined as being Map<String, String>. So, the type declaration here was wrong before (and the analyzer actually complained about it in strong mode). That's why I changed the declaration. As you pointed out, this is actually not JSON. So, I changed the name to the generic "params", which is what the invokeExtension documentation uses for it.

[1] https://www.dartdocs.org/documentation/vm_service_client/0.2.3/vm_service_client/VMIsolateRef/invokeExtension.html